src/components/TopicSelector/TopicSelector.connector.js

```diff
@@ -1,12 +1,21 @@
+import { includes } from 'lodash'
 import { connect } from 'react-redux'
 import {
   findTopics,
   clearTopics,
-  getTopicResults
+  getTopicResults,
+  getTopicsSearchTerm
 } from '../HyloEditor/HyloEditor.store'
 
 export function mapStateToProps (state, props) {
+  // TODO: move this into a store file somewhere... but where?
```

Comment:
Personally I'd be prone to moving all the topics graphql, actions, selectors, and reducer into the root as discrete files in their respective homes. I know what is lost in that, but the kind of SOC which is gained I (lightly) maintain will serve us better over time.